### PR TITLE
cutover to winget

### PR DESCRIPTION
**Pull request by dhruvchand**

### CODE REVIEW

File: advanced-examples/customization-wsl/DevBox/workload.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 $schema: "1.0"
 name: devbox-customization
 tasks:
-  - name: choco
+  - name: winget
```
Comment (DavidObando) [inline advanced-examples/customization-wsl/DevBox/workload.yaml:4]: I think we're supposed to add a copy of the winget task code here, to replace the choco task that is being deleted.

Comment (dhruvchand) [reply advanced-examples/customization-wsl/DevBox/workload.yaml:4]: great point, I should have added this in the description, but I was going to merge this after we have the intrinsic winget task live - what do you think?

Comment (DavidObando) [reply advanced-examples/customization-wsl/DevBox/workload.yaml:4]: I think having the sample here would be interesting to a lot of people now, so I tend to think adding a copy of the latest winget code would be my preference. I do love the notion of leveraging intrinsic tasks for these samples, we should sync about this soon so we can establish a timeline for it to happen.

Comment (dhruvchand) [reply advanced-examples/customization-wsl/DevBox/workload.yaml:4]: Great point, updated the PR and added a link in task.yaml to point to devcenter-catalog (y)

**Review by DavidObando — APPROVED** [review]

**Pull request merged**